org.opent2t.sample.lamp.superpopular/test/lampTests.js

```diff
@@ -140,24 +101,25 @@ function runLampTests(settings) {
     });
 
     test.serial('GetColourMode', t => {
-        return runTest(t, false, () => {
+        return helpers.runTest(settings, t, () => {
             return OpenT2T.invokeMethodAsync(translator, SchemaName, 'getDevicesColourMode', [deviceId]).then((response) => {
-                t.is(response.rt[0], 'oic.r.mode');
-                t.truthy(Object.prototype.toString.call(response.modes) === '[object Array]', 'Verify modes array returned');
+                // t.is(response.rt[0], 'oic.r.mode');
+                // t.truthy(Object.prototype.toString.call(response.modes) === '[object Array]', 'Verify modes array returned');
+                helpers.verifyModesData(t, response);
```

Comment:
synced offline, good to go

---
In reply to: [103067071](https://github.com/openT2T/translators/pull/185#discussion_r103067071) [](ancestors = 103067071)